test/examples/ucp_client_server.c

```diff
@@ -0,0 +1,450 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2018.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+/*
+ * UCP client - server example utility
+ * -----------------------------------------------
+ *
+ * Server side:
+ *
+ *    UCX_NET_DEVICES=mlx5_0:1 UCX_TLS=rc ./ucp_client_server
```

Comment:
since we plan solve this limitation, i think setting the device/tls vars should be in "Notes" and not in the "Usage " section